### PR TITLE
omega-h: bump version used at BBP 9.31.2.dev2 -> 9.31.3.dev1

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

File: var/spack/repos/builtin/packages/omega-h/package.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class OmegaH(CMakePackage):
     git      = "https://github.com/BlueBrain/omega_h.git"
 
     version('develop', branch='master')
-    version('9.31.2.dev2', sha256='08be63b68a012ed8fcf4b7a217cbaadf46faa3f3c937ec8224ff9c542eb6ae62')
+    version('9.31.3.dev1', sha256='47ccb373b4fb4cc6b56ff932badb5674b0c27ee23ad8f3d8197d49fd7678f0a6')
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/omega-h/package.py:21]: Not a fan.

The old package is still going to hang around in the package installation database. Is there any harm in leaving it in?

Comment (tristan0x) [reply var/spack/repos/builtin/packages/omega-h/package.py:21]: No

**Comment by tristan0x** [comment]
Does the CI fail because of matio download issue ?

```
2:23:32  ==> Fetching from https://sourceforge.net/projects/matio/files/matio/1.5.17/matio-1.5.17.tar.gz failed.
12:25:09  curl: (22) The requested URL returned error: 503
12:25:09  curl: (37) Couldn't open file /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/701/mirror/binaries/matio/matio-1.5.17.tar.gz
12:25:09  curl: (37) Couldn't open file /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/701/mirror/binaries/_source-
```

**Comment by tristan0x** [comment]
> Does the CI fail because of matio download issue ?
> 
> ```
> 2:23:32  ==> Fetching from https://sourceforge.net/projects/matio/files/matio/1.5.17/matio-1.5.17.tar.gz failed.
> 12:25:09  curl: (22) The requested URL returned error: 503
> 12:25:09  curl: (37) Couldn't open file /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/701/mirror/binaries/matio/matio-1.5.17.tar.gz
> 12:25:09  curl: (37) Couldn't open file /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/701/mirror/binaries/_source-
> ```

Should I create a issue to report that the source archive should be cached somehow?

**Comment by tristan0x** [comment]
ping @matz-e and @pramodk

**Comment by matz-e** [comment]
Retest this please

**Comment by pramodk** [comment]
omega-h is already updated few days ago.

**Pull request closed without merging**